### DISCUSSION TITLE
Prometheus高可用自动分区方案 | Qinng

### DISCUSSION THREAD

**Discussion by qingwave** [qingwave/qingwave.github.io#64] [Announcements]
https://qingwave.github.io/prometheus-auto-federation/ 

 在Prometheus分区实践中我们介绍了使用集群联邦与远程存储来扩展Prometheus以及监控数据持久化，但之前的分区方案存在一定不足，如分区配置较难维护，全局Prometheus存在性能瓶颈等，本文通过Thanos+Kvass实现更优雅的Prometheus扩展方案。

**Comment by shiyajuan123** [comment]
哇哦，青蛙老师好棒棒
